apps/game/templates/competition/detail.html

```diff
@@ -8,9 +8,12 @@
             <div class="mirror__item mrgb--">
                 <h1 class="mrgb0">{{ competition }}</h1>
 
-                {% if competition.description %}
-                    <p class="mrgv0 text-muted">{{ competition.description | linebreaksbr }}</p>
-                {% endif %}
+                <ul class="list-inline list-inline--divided text-muted">
+                    <li>{{ competition.sport.icon|safe }} {{ competition.sport }}</li>
+                    {% if competition.description %}
+                        <li>{{ competition.description }}</li>
```

Comment:
I agree, you should then probably change the type of `Competition.description` from `TextField` to `CharField`.